jobs/config.json

```diff
@@ -117,6 +130,13 @@
   ]
 },
 
+"ci-kubernetes-node-kubelet-1.6": {
+  "scenario": "kubernetes_kubelet",
+  "args": [
+    "--branch=release-1.6"
```

Comment:
I don't think that's correct - otherwise I wouldn't need to specify it in `jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-repo.yaml`.